src/gimelstudio/interface/addnode_menu.py

```diff
@@ -153,6 +153,8 @@ def __init__(self, parent, node_registry, size,
 
         self.SetBackgroundColour(const.ADD_NODE_MENU_BG)
 
+        self.Bind(wx.EVT_KILL_FOCUS, self.Dismiss)
```

Comment:
This won't work because ``Dismiss`` won't accept the automatically passed ``event`` parameter. (I realize you didn't see that likely because you didn't have the chance to test it. :slightly_smiling_face:)

Probably best to use the solution @POMXARK suggested, which works well from my tests on Zorin 17. 
```
        self.Bind(wx.EVT_KILL_FOCUS, self.ClickOut)
            
    def ClickOut(self, event):
        self.Dismiss()
```